src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
@DamianEdwards If you run a meta tag with a class attribute through w3c doesn't like it:

**Input:**
```HTML
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"><!-- <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"> -->
<html>
<head>
	<title>Test</title>
	<meta name="hello" content="world" class="foo" />
</head>
<body>
</body>
</html>
```

**Error related to meta tag:**
![image](https://cloud.githubusercontent.com/assets/2008729/10901453/82427bf4-81a3-11e5-95f5-b8d22e6f908a.png)